modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -186,18 +186,35 @@ kind: ConfigMap
 metadata:
   name: dpdk-checkup-config
 data:
-  spec.timeout: 1h2m
+  spec.timeout: 10m
   spec.param.NetworkAttachmentDefinitionName: "dpdk-network-1"
-  spec.param.trafficGeneratorImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:v0.1.1"
-  spec.param.vmContainerDiskImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:v0.1.1"
-  status.succeeded: true
-  status.failureReason: " "
-  status.startTimestamp: 2022-12-21T09:33:06+00:00
-  status.completionTimestamp: 2022-12-21T11:33:06+00:00
-  status.result.actualTrafficGeneratorTargetNode: worker-dpdk1
-  status.result.actualDPDKVMTargetNode: worker-dpdk2
-  status.result.dropRate: 0
-----
+  spec.param.trafficGenContainerDiskImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:v0.1.1"
+  spec.param.vmUnderTestContainerDiskImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:v0.1.1"
+  status.succeeded: "true" <1>
+  status.failureReason: "" <2>
+  status.startTimestamp: "2023-07-31T13:14:38Z" <3>
+  status.completionTimestamp: "2023-07-31T13:19:41Z" <4>
+  status.result.trafficGenSentPackets: "480000000" <5>
+  status.result.trafficGenOutputErrorPackets: "0" <6>
+  status.result.trafficGenInputErrorPackets: "0" <7>
+  status.result.trafficGenActualNodeName: worker-dpdk1 <8>
+  status.result.vmUnderTestActualNodeName: worker-dpdk2 <9>
+  status.result.vmUnderTestReceivedPackets: "480000000" <10>
+  status.result.vmUnderTestRxDroppedPackets: "0" <11>
+  status.result.vmUnderTestTxDroppedPackets: "0" <12>
+----
+<1> Specifies if the checkup is successful (`true`) or not (`false`).
+<2> The reason for failure if the checkup fails.
+<3> The time when the checkup started, in RFC 3339 time format.
+<4> The time when the checkup is complete, in RFC 3339 time format.
```

Comment:
`startTimestamp` uses past tense "started" while - `completionTimestamp` uses present tense "is complete". could you align `completionTimestamp` to be past, as the other params?

```suggestion
<4> The time when the checkup has completed, in RFC 3339 time format.
```

Reply:
Done.